### PR TITLE
renames a single achievement

### PR DESCRIPTION
**Pull request by BootlegBow**
## About The Pull Request

Renames the 'Rule 8' achievement to 'Rule 3'
## Why It's Good For The Game

"Our rule 8 is about mutiny. That achievement has nothing to do with mutiny. Please don't mutiny in an attempt to get that achievement." - some admin to an idiot last week

## Changelog
:cl:
tweak: renames rule 8 to rule 3
/:cl:

### CODE REVIEW

**Comment by EgorDinamit** [comment]
Goddamn /tg/ and their "kissing improvements PR".

File: code/datums/achievements/misc_achievements.dm [hunk 1]
```diff
@@ -84,11 +84,11 @@
 	desc = "How does it feel to know that your workplace values a mop bucket on wheels more than you?" // i can do better than this give me time
 	database_id = MEDAL_CLEANBOSS
 
-/datum/award/achievement/misc/rule8
-	name = "Rule 8"
+/datum/award/achievement/misc/rule3
+	name = "Rule 3"
```
Comment (EgorDinamit) [inline code/datums/achievements/misc_achievements.dm:88]: Technically all you have to do is change the name, to avoid further merge conflicts.

Comment (BootlegBow) [reply code/datums/achievements/misc_achievements.dm:88]: So you're saying to leave everything but the name as rule 8?

Comment (EgorDinamit) [reply code/datums/achievements/misc_achievements.dm:88]: Yep. It doesn't matter to players what's in the code.

Comment (BootlegBow) [reply code/datums/achievements/misc_achievements.dm:88]: Alright

**Pull request merged**